youtube_dl/extractor/medialaan.py

```diff
@@ -140,7 +140,7 @@ def _real_extract(self, url):
                 '\\\\', '\\').replace(r'\"', '"').replace(r"\'", "'"))
 
         vod_id = config.get('vodId') or self._search_regex(
-            (r'\\"vodId\\"\s*:\s*\\"(.+?)\\"',
+            (r'\"vodId\"\s*:\s*\"(.+?)\"',
```

Comment:
1. Keep the old regex as well.
2. No need for escaping it's already a raw string.

Reply:
Thanks for the feedback, have fixed it.